internal/controllers/node_label_module_version_reconciler.go

```diff
@@ -192,6 +211,7 @@ func (nlmvha *nodeLabelModuleVersionHelper) reconcileLabels(modulesLabels map[st
 // validity is checked by verifying that devicePlugin pod defined
 // by name, namespace,role is missing. In case the pod is present, no matter in
 // what state, then the action is invalid
+/*
```

Comment:
Can we remove that commented out function?

Reply:
fixed